GLAA.Domain/GLAAContext.cs

```diff
@@ -46,18 +48,18 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             }
 
             // many-to-many
-            modelBuilder.Entity<LicenceCountry>()
-                .HasKey(x => new { x.LicenceId, x.CountryId });
+            modelBuilder.Entity<LicenceWorkerCountry>()
```

Comment:
Is there no longer a need to have a country assigned to a licence (the operating country for example)? or is that handled elsewhere?

Reply:
It's a straight rename of the class, the functionality is still there 😄